schedule/yast/sle/usb_install.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: USBinstall
+description: >
+  Basic installation test with USB boot. Verification that usb repo is enabled
+  after first boot. Test covers both Online and Full medium installations.
+vars:
+  DESKTOP: textmode
+  USBBOOT: 1
+  YUI_REST_API: 1
+schedule:
+  system_preparation:
+    - console/system_prepare
+    - console/hostname
+    - console/force_scheduled_tasks
+    - shutdown/grub_set_bootargs
+    - console/enable_usb_repo
```

Comment:
I guess this one fits better in validation section than in preparation.

Reply:
I would group in `system_preparation` the typical test module we schedule there:
```
- console/system_prepare
- console/hostname
- console/force_scheduled_tasks
```
we could remove `shutdown/grub_set_bootargs` as we are not booting this installation again.
 Then for system validation we would have the bare minimum:
 `console/enable_usb_repo`.